namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/rds.tf

```diff
@@ -32,3 +32,38 @@ resource "kubernetes_secret" "hmpps_interventions_rds" {
     url                   = "postgres://${module.hmpps_interventions_rds.database_username}:${module.hmpps_interventions_rds.database_password}@${module.hmpps_interventions_rds.rds_instance_endpoint}/${module.hmpps_interventions_rds.database_name}"
   }
 }
+
+module "hmpps_interventions_split_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.16.10"
+  cluster_name           = var.cluster_name
+  team_name              = var.team_name
+  business-unit          = var.business_unit
+  application            = var.application
+  is-production          = var.is_production
+  namespace              = var.namespace
+  environment-name       = var.environment
+  infrastructure-support = var.infrastructure_support
+  rds_family             = var.rds_family
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "hmpps_interventions_split_rds" {
+  metadata {
+    name      = "postgres"
```

Comment:
This would override the `postgres` secret defined by the `hmpps_interventions_rds` resource. Can we use another name, please?

Reply:
I have renamed it now